components/private/Snapshot/src/SnapshotTestCase/MDCSnapshotTestCase.m

```diff
@@ -125,4 +130,25 @@ - (NSString *)getDeviceName {
   return deviceName;
 }
 
+- (void)changeViewLayoutToRTL:(UIView *)view {
+  view.semanticContentAttribute = UISemanticContentAttributeForceRightToLeft;
+  for (UIView *subview in view.subviews) {
+    [self changeViewLayoutToRTL:subview];
+  }
+}
+
+- (void)changeTextInputToRTL:(UIView *)view {
+  if ([view conformsToProtocol:@protocol(UITextInput)]) {
+    id<UITextInput> textInput = (id<UITextInput>)view;
+    UITextRange *textRange = [textInput textRangeFromPosition:textInput.beginningOfDocument
+                                                   toPosition:textInput.endOfDocument];
+    if (textRange) {
+      [textInput setBaseWritingDirection:UITextWritingDirectionRightToLeft forRange:textRange];
+    }
+  }
+  for (UIView *subview in view.subviews) {
```

Comment:
Should we just call `[self changeViewLayoutToRTL:]` here?

Reply:
These two helpers are separate to make them more clear. Since it is only used in testing, I am not too worried about the performance.